sotodlib/wiregrid/calibrate.py

```diff
@@ -0,0 +1,404 @@
+#!/usr/bin/python3
+
+# import modules
+# python standard
+import os
+import sys
+import scipy
+from scipy import odr
+import numpy as np
+from iminuit import cost, Minuit
+from matplotlib.patches import Ellipse, Circle
+
+# SO specific
+from sotodlib import core
+from sotodlib.core import Context
+from sotodlib.io import hk_utils
+import sotodlib.io.load_smurf as ls
+from sotodlib.io.load_smurf import Observations
+
+wg_degpercount = 360/52000
+stream_ids = ['ufm_mv19', 'ufm_mv18', 'ufm_mv22', 'ufm_mv29', 'ufm_mv7', 'ufm_mv9', 'ufm_mv15'] # e.g. MF1
+
+def get_det_angle(aman, **kwargs):
```

Comment:
Typically keyword arguments are passed in this way if you want to pass a set of optional arguments to a function called inside of this function. For the default parameters for this function you should have them exposed explicitly (i.e. wg_config, wg_offset, stopped_time)

Reply:
Thanks. I will put all arguments explicitly.